src/gui/transferlistwidget.h

```diff
@@ -55,7 +55,7 @@ class TransferListWidget: public QTreeView
     Q_OBJECT
 
 public:
-    TransferListWidget(QWidget *parent, MainWindow *main_window);
+    TransferListWidget(QWidget *parent, MainWindow *m_mainWindow);
```

Comment:
What about different param name in prototype and definition, what are qbt conventions?

Reply:
The naming in the code is incorrect, of course. Thanks for pointing this out. Has to be `mainWindow`.

Reply:
Oops... I didn't notice this drawback. m_ is for member variables only, of course.